### PR TITLE
add sfold

### PR DESCRIPTION
**Pull request by reytakop**

### CODE REVIEW

**Comment by nilchia** [comment]
This will pass the conda build test but still, the PATHs are not set correctly

**Comment by nilchia** [comment]
Now sfold is installed but the problem is in `build.sh`  when:
```
cp -r bin/* ${PREFIX}/bin
cp -r lib/* ${PREFIX}/lib
```
the perl and r-base are removed from bin/ so sfold can not find it.

However when I install the package with
```
conda create -n test perl r-base
conda activate test
conda install sfold-2.2-pl5262r35h9bf148f_0.tar.bz2
```
it works fine

**Comment by nilchia** [comment]
Still all dependencies are removed from `/bin`
Don't know why

**Comment by reytakop** [comment]
@bgruening Thanks!

File: recipes/sfold/LICENSE [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+9. Notices
+9.1 Except as otherwise expressly provided in this Agreement, all notices and other communications in connection with this Agreement shall be in writing and shall be sent by postage prepaid certified mail return receipt requested, overnight delivery service or fax to Health Research Inc. at the address set forth below, and to You, at the email address you provided to Health Research Inc. with Your application for this license, or to such other addresses as may be designated by either Party by like notice:
+To Health Research Inc. at 150 Broadway Suite 560, Menands, New York 12204
```
[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/sfold/LICENSE:70]: Please don't make recommendations in LICENSE files.

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by nilchia** [comment]
I tested the artifacts.
It works!

**Pull request merged**